backend/forms/forms.py

```diff
@@ -2,7 +2,45 @@
 from wtforms import (StringField, IntegerField, DateField,
                      HiddenField, PasswordField, SelectField,
                      SubmitField)
-from wtforms.validators import DataRequired, Email, Optional, Length, Regexp
+from wtforms.validators import (DataRequired, Email, Optional,
+                                Length, Regexp, ValidationError)
+
+from app import db
+from models.users import User
+
+
+class Check_users_field(object):
+
+    """Custom validator.
+
+    Validate for unique field value.
+    Skips record in database with current user's id.
+
+    """
+
+    def __init__(self, model, property_to_find, message=None):
+        if not message:
+            message = "This field's value is already exists in database."
+        self.message = message
+        self.model = model
+        self.property_to_find = property_to_find
+
+    def __call__(self, form, field):
+        if not form.id.data:
+            form.id.data = None
+        self.query = db.session.query(self.model).filter(
```

Comment:
It doesn't make sense to store result in validator's class instance, local variable is enough.